### ISSUE TITLE
ion-scroll scrollY blocked on Android devices

### ISSUE THREAD

**Issue by peterpeterparker** [driftyco/ionic#11344]
**Ionic version:**  (check one with "x")
[X ] **3.x**

**I'm submitting a ...**  (check one with "x")
[X ] bug report

**Current behavior:**
Using list in ion-scroll (scrollY) on android devices where both click and swipe are used, make the list not scrollable (see screenshot to see which part of the screen isn't scrollable).

**Expected behavior:**
A working scrollable list in an ion-scroll with both click and swipe support.

**Steps to reproduce:**

1. Here a project to reproduce the problem https://github.com/peterpeterparker/myIonScroll
2. ionic platform add android
3. ionic build android --prod
4. start and test on a device. the scroll list on home.html not gonna be scrollable

5. if you wish, you could remove the swipe event from the list, redo the test behavior and will notice that without it the list gonna be scrollable

**Related code:**

See https://github.com/peterpeterparker/myIonScroll/blob/master/src/pages/home/home.html

**Other information:**

I'm freaking stuck with the upgrade to Ionic 3 because of such scroll problems since more than two weeks now, not that I deserve an award, just to let you know that I tried and invested hours to solve these. 

I resolved the similar problem with iOS (basically, a foreign lib which was using hammerjs which seems to be buggy has that effect, see https://forum.ionicframework.com/t/swipe-event-freeze-scroll-on-ios-and-android/86085/12).

But in this issue, I don't import any foreign libraries.

Same project I provided as example works (= list is scrollable) in chrome debug, in iOS thru Xcode but not on an android device.

I don't use Crosswalk.

**Ionic info:** (run `ionic info` from a terminal/cmd prompt and paste output below):

Cordova CLI: 6.5.0 
Ionic Framework Version: 3.0.1
Ionic CLI Version: 2.2.1
Ionic App Lib Version: 2.2.0
Ionic App Scripts Version: 1.3.4
ios-deploy version: 1.9.0 
ios-sim version: 5.0.13 
OS: macOS Sierra
Node Version: v7.2.1
Xcode version: Xcode 8.3 Build version 8E162

<img width="378" alt="capture d ecran 2017-04-19 a 16 40 51" src="https://cloud.githubusercontent.com/assets/16886711/25351638/84359a48-2929-11e7-9263-338d947354c5.png">

**Comment by jgw96** [comment]
Hello! Thanks for opening an issue with us. I am going to close this issue as a duplicate of https://github.com/driftyco/ionic/issues/11112. Please feel free to add your findings to that issue.

**Comment by peterpeterparker** [comment]
@jgw96 could we hope to see a fix in a near future?
